apps/data-pipeline/ap-exams-importer/src/data.ts

```diff
@@ -0,0 +1,737 @@
+// https://apstudents.collegeboard.org/courses
+// https://catalogue.uci.edu/informationforprospectivestudents/undergraduateadmissions/#advancedplacementandinternationalbaccalaureatecredittext
+
+import type { APCoursesGrantedTree } from "@packages/db/schema";
+
+export const geCategories = [
+  "GE-1A",
+  "GE-1B",
+  "GE-2",
+  "GE-3",
+  "GE-4",
+  "GE-5A",
+  "GE-5B",
+  "GE-6",
+  "GE-7",
+  "GE-8",
+] as const;
+
+type APExam = {
+  catalogueName?: string;
+  rewards: {
+    acceptableScores: (1 | 2 | 3 | 4 | 5)[];
```

Comment:
are 1 and 2 really necessary considering that for all exams no credit is granted for scores below a 3?

Reply:
this field is meant to model an exam score in general; it's theoretically possible that scores 1 and 2 were mentioned